File: saizad-mvvm/src/main/java/com/saizad/mvvm/delegation/fragment/FragmentAppLifecycleCallBack.java
package com.saizad.mvvm.delegation.fragment;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;

import com.saizad.mvvm.delegation.BaseLifecycleCallBack;

public interface FragmentAppLifecycleCallBack extends BaseLifecycleCallBack {
    void onViewCreated(View view, @Nullable Bundle savedInstanceState, boolean recycled);
}

File: saizad-mvvm/src/main/java/com/saizad/mvvm/delegation/fragment/FragmentAppLifecycleCallBack.kt
package com.saizad.mvvm.delegation.fragment

import android.os.Bundle
import android.view.View
import com.saizad.mvvm.delegation.BaseLifecycleCallBack

interface FragmentAppLifecycleCallBack : BaseLifecycleCallBack {
    fun onViewCreated(view: View, savedInstanceState: Bundle?, recycled: Boolean)
    fun setHasOptionsMenu(hasOptionsMenu: Boolean)
    fun onBackPressed(): Boolean
}